pkgs/development/python-modules/fitparse/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+}:
+buildPythonPackage rec {
+  pname = "fitparse";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "fitparse";
+    hash = "sha256-LWkQIkUt6m2rrRPMbgF8pGf+ijqJXNOsZ6UKe7cWtKk=";
+  };
```

Comment:
fetchFromGitHub is preferred nowadays unless there is a compelling reason to not use it.

Reply:
Had no idea, done.